class.jetpack.php

```diff
@@ -6806,6 +6806,7 @@ public static function get_jetpack_options_for_reset() {
 	 * @return string | url without the guff
 	 */
 	public static function build_raw_urls( $url ) {
+		jetpack_deprecated_function( __METHOD__, 'jetpack_get_site_suffix', '9.2' );
```

Comment:
Any reason not to remove the logic here and have `build_raw_urls` call `jetpack_get_site_suffix`?

Reply:
Duh! Fixed in fd46f6a

Reply:
fd46f6a37ff004d66bf58b776085872391a911ad